recipe/build.sh

```diff
@@ -31,6 +31,7 @@ cmake -B scipoptsuite-build -S "${SRC_DIR}/scipoptsuite" \
       -D READLINE=OFF \
       -D SYM=bliss \
       -D EXPRINT=cppad \
-      -D CLIQUER=ON
+      -D CLIQUER=ON \
+      -D CMAKE_INSTALL_LIBDIR=$PREFIX/lib
```

Comment:
```suggestion
      -D CMAKE_INSTALL_LIBDIR=lib
```
We use relative, the absolute part is handled by `CMAKE_INSTALL_PREFIX` or `--prefix`, but really I would recommend using `${CMAKE_ARGS}` as it contains (and evolves) other conda-forge relevant variables.